addons/survey/data/survey_demo_conditional.xml

```diff
@@ -4,7 +4,7 @@
     <record id="survey_demo_burger_quiz" model="survey.survey">
         <field name="title">Burger Quiz</field>
         <field name="access_token">burger00-quiz-1234-abcd-344ca2tgb31e</field>
-        <field name="user_id" ref="base.user_demo"/>
+        <field name="user_ids" eval="[(4, ref('base.user_demo'))]"/>
```

Comment:
Would say it is an open one. Would have between 0 or 1 "hidden" survey, demo are there to be shown.